MonoidalCategories/PackageInfo.g

```diff
@@ -10,7 +10,7 @@ SetPackageInfo( rec(
 
 PackageName := "MonoidalCategories",
 Subtitle := "Monoidal and monoidal (co)closed categories",
-Version := "2022.01-02",
+Version := "2022.02-08",
```

Comment:
I oversaw this. It should be `2022.02-01`. The same issue in `CategoryConstructor`.

Reply:
This allows us to have two or more releases per day.